packages/react-core/src/components/ToggleGroup/examples/ToggleGroupCompact.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { ToggleGroup, ToggleGroupItem } from '@patternfly/react-core';
+
+export const ToggleGroupCompact: React.FunctionComponent = () => {
+  const [state, setState] = React.useState({
+    isSelected: {
+      'compact-1': false,
+      'compact-2': false
+    }
+  });
+  const handleItemClick = (isSelected, event) => {
```

Comment:
The parameters here just need to be typed (this also applies to all of the example files that have the handleItemClick method)